Create new MKW_Classes.lua file, update core and pointers

vabold opened the pull request:
This commit aims to create Lua structures for various MKW classes. This will become the new standard for future script creating/core updates.

- [ ] Verify contents of MKW_Classes.lua
- [ ] Update MKW_Core.lua
- [x] Update MKW_Pointers.lua
- [ ] Update scripts to account for new files

vabold committed:
Merge new base for pointers (#28)

* Fix Guaranteed Crashes on Startup

This removes outdated names and replaces them with new ones. The acceleration functions will be rewritten in the future, so functionality does not matter for them at the moment.

* MKW_Pointers Formatting Update

Suggested by @malleoz.

* Fix Inconsistent Formatting

Co-authored-by: xi <energeticcamel55@gmail.com>
Co-authored-by: xi <104683822+ximk@users.noreply.github.com>

vabold committed:
Add new entries and restructure MKW_Pointers.lua

This commit will break all Lua scripts, including MKW_Core.lua. Future commits will resolve these issues.

LordDraXula commented:
Untested, but Pointers fit the new coordination system.
Merge to be made once all the necessary scripts were updates.

ᵖˢ ᵈᵒⁿᵗ ʷᵒʳʳʸ ᵃᵇᵒᵘᵗ ᵗʰᵉ ʳᵉᵛᶦᵉʷ ᵗʰᶦⁿᵍʸ

vabold commented:
No complaints seem to be made about MKW_Pointers. Marking task as complete.

vabold committed:
Create MKW_Classes.lua

This code was blitzed out in two days and is not tested in any way. This will need heavy review before proceeding.

vabold committed:
Initialize subtables correctly

vabold committed:
Fix wheel subtables in Player

vabold committed:
Fix hudPlayers subtable in RacedataSettings
